packages/admin/cms-admin/src/pages/pageTree/PageLabel.tsx

```diff
@@ -28,12 +33,26 @@ const PageLabel: React.FunctionComponent<PageLabelProps> = ({ page, disabled, on
                     />
                 )}
             </LinkText>
+
+            {documentType?.infoTag && page.document?.__typename && (
+                <InfoPanel style={{ padding: 0 }} label={documentType?.infoTag({ __typename: page.document.__typename, ...page.document })} />
+            )}
         </Root>
     );
 };
 
 export default PageLabel;
 
+const InfoPanel = styled(Chip)`
+    margin-left: auto;
+    margin-right: 10%;
+    height: 20px;
+
+    span {
```

Comment:
@jamesricky is there a better way to change the padding of the Chip's content?

Reply:
Otherwise you could probably access the label like this:

```tsx
import withStyles from "@mui/styles/withStyles";

const InfoPanel = withStyles({
    root: {
        marginLeft: "auto",
        marginRight: "10%",
        height: 20,
    },
    label: {
        padding: "0 5px",
    },
})(Chip);
```

Reply:
didn't realize that the chip has a prop for size, changed it